pennylane_lightning_gpu/src/tests/Test_StateVectorCudaManaged_Param.cpp

```diff
@@ -20,6 +20,385 @@
 using namespace Pennylane;
 using namespace CUDA;
 
+TEMPLATE_TEST_CASE("LightningGPU::applyPauliRot RX", "[LightningGPU_Param]",
+                   double) {
+    using cp_t = std::complex<TestType>;
+    const size_t num_qubits = 1;
+    SVDataGPU<TestType> svdat{num_qubits};
+
+    const std::vector<TestType> angles{{0.1}, {0.6}};
+    std::vector<std::vector<cp_t>> expected_results{
+        std::vector<cp_t>{{0.9987502603949663, 0.0},
+                          {0.0, -0.04997916927067834}},
+        std::vector<cp_t>{{0.9553364891256061, 0.0}, {0, -0.2955202066613395}},
+        std::vector<cp_t>{{0.49757104789172696, 0.0}, {0, -0.867423225594017}}};
+
+    std::vector<std::vector<cp_t>> expected_results_adj{
+        std::vector<cp_t>{{0.9987502603949663, 0.0},
+                          {0.0, 0.04997916927067834}},
+        std::vector<cp_t>{{0.9553364891256061, 0.0}, {0, 0.2955202066613395}},
+        std::vector<cp_t>{{0.49757104789172696, 0.0}, {0, 0.867423225594017}}};
```

Comment:
Could you please let me know how did you get these numbers? Thanks

Reply:
Thank you @multiphaseCFD for the review! All of these results are calculated by Pennylane. I can share the python script with you.